x/programs/runtime/dependencies.go

```diff
@@ -25,6 +27,6 @@ type Runtime interface {
 }
 
 type Storage interface {
```

Comment:
maybe Storage name is too generic been back and forth on that but we should probably make it less vauge.